core/src/main/java/dev/streamx/cli/ParameterExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+package dev.streamx.cli;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import java.io.PrintWriter;
+import org.jboss.logging.Logger;
+import picocli.CommandLine;
+import picocli.CommandLine.IParameterExceptionHandler;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.ParameterException;
+import picocli.CommandLine.UnmatchedArgumentException;
+
+@ApplicationScoped
+public class ParameterExceptionHandler implements IParameterExceptionHandler {
+
+  @Inject
+  Logger log;
+
+  @Override
+  public int handleParseException(ParameterException ex, String[] args) {
+    log.error("Parameter exception occurred.", ex);
+
+    CommandLine cmd = ex.getCommandLine();
+    PrintWriter writer = cmd.getErr();
+
+    writer.println(ex.getMessage());
+    UnmatchedArgumentException.printSuggestions(ex, writer);
+    writer.print(cmd.getHelp().fullSynopsis());
+
+    CommandSpec spec = cmd.getCommandSpec();
+    writer.printf("Try '%s --help' for more information.%n", spec.qualifiedName());
```

Comment:
If it's possible - add newline to separate suggestion "Try" from rest of information.

Reply:
Done